akd/src/ecvrf/mod.rs

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
So that whole folder has been deleted, so I'm guessing that they aren't using VRFs anywhere anymore. I think we're on our own sync-wise, but this does follow the spec pretty clearly so unless the RFC changes we should be good.

Reply:
Apparently they were removed due to lack of use. Great find!